vpr/src/pack/re_cluster_util.cpp

```diff
@@ -82,7 +82,7 @@ void commit_mol_move(const ClusterBlockId& old_clb,
         g_vpr_ctx.mutable_placement().block_locs.resize(g_vpr_ctx.placement().block_locs.size() + 1);
         get_imacro_from_iblk(&imacro, old_clb, g_vpr_ctx.placement().pl_macros);
         set_imacro_for_iblk(&imacro, new_clb);
-        place_one_block(new_clb, device_ctx.pad_loc_type, NULL);
+        place_one_block(new_clb, device_ctx.pad_loc_type, NULL, NULL);
```

Comment:
@MohamedElgammal  can you briefly tell me what does place_one_block do in here? Does this only care to find a location for a specific block? Right now, place_one_block uses the centroid_placement as the primary initial placement algorithm. Does it create any problem here? We can directly call place_macro instead of place_one_block.

Reply:
@saaramahmoudi I am calling this function to find a place for a new cluster --if created. If the place_one_block function is now running centroid placement, I agree that we can either try to place the new block in its centroid location (keep using place_one_block) or just place it in a random location using place_macro directly.

Reply:
@MohamedElgammal Is it okay if I change this to place_macro?